content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,264 @@
+    class na blue_on_white;
+    class k8s,cp white_on_blue;
+{{</ mermaid >}}
+
+### Autoscaler implementations
+
+Cluster autoscaler and Karpenter are the two Node autoscalers currently supported by
+[sig-autoscaling](https://github.com/kubernetes/community/tree/master/sig-autoscaling).
+
+From the perspective of a cluster user, both autoscalers should provide a similar Node autoscaling
+experience. Both will provision new Nodes for unschedulable Pods, and both will
+consolidate the Nodes that are no longer optimally utilized.
+
+The most of the difference between the autoscalers is in how they're set up and
+configured, as well as in how well they're integrated with a given cloud provider.
+The autoscalers also provide features outside the Node autoscaling scope described
+on this page, and the features differ between them.
+
+Consult the sections below, and the linked documentation for the individual
+autoscalers to decide which autoscaler fits your use-case better.
+
+#### Cluster Autoscaler
+
+Cluster Autoscaler adds or removes Nodes to pre-configured Node groups.
+Node groups generally map to some sort of provider-side VM group. A
+single instance of Cluster Autoscaler can simultaneously manage multiple
+Node groups. When provisioning, Cluster Autoscaler will add Nodes to the group
+that best fits the requests of pending Pods. When consolidating, Cluster Autoscaler
+always selects specific Nodes to remove as opposed to just resizing the underlying VM group.
+
+Main differences vs Karpenter:
+* Cluster Autoscaler (CA) provides features related to just Node autoscaling. Node lifecycle
+  beyond that is not in its scope.
+* CA doesn't support auto-provisioning, the Node groups it can provision from have to be
+  pre-configured.
+
+Additional context:
+* Documentation overview: https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/README.md
+* Supported cloud providers: https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/README.md#faqdocumentation
+* Extensive FAQ: https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md
+* Contact: https://github.com/kubernetes/community/tree/master/sig-autoscaling#contact
+
+#### Karpenter
+
+Karpenter auto-provisions VMs based on [NodePool](https://karpenter.sh/docs/concepts/nodepools/)
+configuration provided by the user. Karpenter handles all aspects of node lifecycle, not just autoscaling.
+It works directly with VMs and doesn't rely on provider-side VM groups.
+
+Main differences vs Cluster Autoscaler (CA):
+* In addition to Node autoscaling, Karpenter provides features intended for managing
+  Node lifecycle altogether (for example grouping Nodes into pools, handling Node upgrades).
+* Karpenter supports auto-provisioning, the user only has to configure a set of
+  constraints for the provisioned Nodes instead of pre-configuring Node groups explicitly.
+
+Additional context:
+* Documentation: https://karpenter.sh/
```

Comment:
This documentation is still not very cloud neutral. Its the best we have now but would be good to revisit this later 

cc: @jonathan-innis, @njtran, @dtzar

Reply:
I think @chrisnegus and @dtzar are tackling this, we should definitely update this link with however this effort lands

Reply:
This is already being worked on. Personally, I don't think we need to have a TODO here, unless we think that it'll actually help make sure this is updated.

Reply:
Also, I'd imagine that we may end up using the same domain down the line anyways since I don't imagine that we want to shift all of the current users away from the current domain, just extend it

Reply:
We are working on making most of the Karpenter.sh site generic, with separate, well-defined sections for supported cloud providers.